pom.xml

```diff
@@ -9,7 +9,7 @@
     <relativePath />
   </parent>
   <artifactId>applitools-eyes</artifactId>
-  <version>1.14-SNAPSHOT</version>
+  <version>${revision}${changelist}</version>
```

Comment:
It looks like you've missed updating the version format for CD release?
https://www.jenkins.io/doc/developer/publishing/releasing-cd/

while it says optional you do need to pick one

Reply:
@timja I pushed the change. I selected to work with revision.